test/v3/connectors/SubmitFinalDeclarationConnectorSpec.scala

```diff
@@ -32,7 +32,7 @@ class SubmitFinalDeclarationConnectorSpec extends ConnectorSpec {
   val request: SubmitFinalDeclarationRequest = SubmitFinalDeclarationRequest(
     Nino(nino),
     taxYear,
-    calculationId
+    CalculationId(calculationId)
```

Comment:
We should update Nino and CalculationID to follow the same format as taxYear and taxYear i.e: `val calculationId: CalculationId = CalculationId("a1e8057e-fbbc-47a8-a8b4-78d9f015c253")`

```suggestion
      calculationId
```